packages/core/src/runtime/resolveResetStyleRules.ts

```diff
@@ -0,0 +1,122 @@
+import hashString from '@emotion/hash';
+import { convertProperty } from 'rtl-css-js/core';
+
+import { RESET_HASH_PREFIX } from '../constants';
+import { GriffelStyle, GriffelResetStyle } from '../types';
+import { compileCSSRules, normalizePseudoSelector } from './compileCSS';
+import { isMediaQuerySelector } from './utils/isMediaQuerySelector';
+import { isLayerSelector } from './utils/isLayerSelector';
+import { isNestedSelector } from './utils/isNestedSelector';
+import { isSupportQuerySelector } from './utils/isSupportQuerySelector';
+import { isObject } from './utils/isObject';
+import { hyphenateProperty } from './utils/hyphenateProperty';
+
+/**
+ * @internal
+ */
+export function createStringFromStyles(styles: GriffelResetStyle) {
+  let ltrCSS = '';
+  let rtlCSS = '';
+
+  // eslint-disable-next-line guard-for-in
+  for (const property in styles) {
+    const value = styles[property as keyof GriffelStyle];
+
+    // eslint-disable-next-line eqeqeq
+    if (value == null) {
+      continue;
+    }
+
+    if (typeof value === 'string' || typeof value === 'number') {
+      const { key: rtlProperty, value: rtlValue } = convertProperty(property, value);
+
+      ltrCSS += `${hyphenateProperty(property)}:${value};`;
+      rtlCSS += `${hyphenateProperty(rtlProperty)}:${rtlValue};`;
+
+      continue;
+    }
+
+    if (property === 'animationName') {
+      // TODO: handle animations
+      throw new Error();
```

Comment:
Shouldn't this be encapsulated inside a `process.env.NODE_ENV !== 'production'`? we're trying to avoid throwing things in production...

Also, empty Error? 😅

Reply:
Let's track this in a follow up issue

Reply:
I was sure that I added support for it 🙁 But... #239.